### ISSUE TITLE
Nuscenes dataset selection for downstream task of semantic segmentation

### ISSUE THREAD

**Issue by nelson-wpwang** [valeoai/SLidR#21]
Hi, I am trying to understand how the fine-tuned model is evaluated. From what I understand, the config["training"] parameter is set to "parametrize", so in downstream.py, the first part fine-tuning would use the 600 scenes in training set of Nuscenes divided by "dataset-skip-ratio", and evaluate the validation loss of model using the 100 custom split. In the second part of downstream, after fine-tuning the model, because the parameter "training" is still set to "parametrize", so the phase is set as "verifying", which will still use the 100 custom split for evaluation. If we want to evaluate the 150 scenes provided by Nuscenes, we have to switch the config["training"] to "validate", and that is going to re-run the fine-tuning with 700 scenes(all training in Nuscenes) divided by skip_ratio.
My question is, isn't this switching is going to change the downstream model? Or is it when you set config["training"] = "parameterize" is just to find the best parameters for the model, and you use config["training"] = "validate" to collect the results you present in the paper?

Thank you very much.

**Comment by CSautier** [comment]
Hi,
In the paper in the ablation study the downstream are done on the "parametrize" sets that we call the mini-train, mini-val splits (table 1, 5 and 6) but the "final" results are done on the official train/val sets from nuscenes (table 2 and 3). Also when we did our parameters search we used the mini-train and mini-val sets as well. The code we provide here by default with the config semseg_nuscenes.yaml reproduces the 1% Finetuning (the 38.3 in our table 2) on the real validation set. "parametrize" should not appear in the downstream config.
Tell me if that answers your questions.
